docs/source/noaa_csp.rst

```diff
@@ -40,160 +51,252 @@ Login to the NOAA CSP
-2. A short description of the instance, i.e., ``This instance supports
-   this <task name> task.``
-3. Tag(s) describing and identifying the respective instance. These
-   allow for improved bookkeeping, especially when a user has multiple
-   or concurrent instance types.
-
-Next, the cluster is defined as shown in the following illustration.
+   
+Please provide a unique name (box pointer by the red-arrow) for the instance.
+Best practices suggest one that is clear, concise, and relevant to the application.
+Click :red-text:`Fork` (in the red-circle) to fork an instance.
 
 .. image:: _static/noaacsp_instance_3.png
 
-The NOAA Parallel Works initiative currently provides 2 CSPs for the
-global-workflow; **AWS** (Amazon Web Services) and **Azure**
-(Microsoft Azure). Existing clusters may also be modified. However
-this is neither recommended or supported.
+Now, an instance is forked, and it is time to configure the cluster.
+
+1. Select a "Recource Account", usually it is: NOAA AWS Commercial Vault.
+2. Select a "Group". Which will be something like: ca-epic, ca-sfs-emc, etc.
+3. Copy and paste you Public key (.ssh/id_rsa.pub, .ssh/id_dsa.pu from your laptop).
+4. Modify "User Bootstrap". If you are not use "ca-epic" group, please UNCOMMENT line 2.
+5. Keep "health Check" as it is.
 
-Finally, when satisfied with the CSP instance configure, click ``Add
-Resource`` as illustrated below.
+Click "Save Changes" at top-right as shown in read circle.
 
 .. image:: _static/noaacsp_instance_4.png
 
+The NOAA ParallelWorks currently provides 3 CSPs:
+**AWS** (Amazon Web Services), **Azure** (Microsoft Azure),
+and **GCP** (Google Cloud Platform).
+Existing clusters may also be modified.
+Better practice is to fork from Marketplace with something similar to your requests.
+
 ******************************
-Configure the NOAA CSP Cluster
+Add CSP Lustre Filesystem
 ******************************
 
-Navigate to the tab and locate the CSP instance configured in the
-previous section and click on the link, `globalworkflowdemo` for this
-example.
+To run global-workflow on CSPs, we need to attached lustre filesystem as run directory.
+Figure, we need to add/define our lustre filesystem, to do so,
+navigate to the middle of noaa.parallel.works web-site left panel and select "Lutre" 
+as pointed by red-arrow, and then click "AddStorage" at top-right as in the red-circle.
 
-.. image:: _static/noaacsp_cluster_1.png
+.. image:: _static/noaacsp_lustre_1.png
 
-The respective CSP cluster maybe then be configured. The mandatory
-configuration attributes are as follows.
+Select `FSx` for AWS FSx lustre filesystem as in red-circle.
+Define the lustre with:
 
-- Availability zone;
-- Disk size and storage type(s);
-- Available compute and resource partitions.
+1. Clear and meaningfull `Resource name` as shown in red-arrow 1
+2. Short sentice for `Decription` as shown in red-arrow 2
+3. Choose **linux** for `Tag` as shown in red-arrow 3
```

Comment:
```suggestion
#. A clear and meaningful `Resource name` as shown by the first red arrow
#. A short sentence for `Description`, as shown in the second red arrow
#. Choose **linux** for `Tag` as shown by red arrow #3
```

Reply:
The # symbol should automatically generate a list. You can keep it the way you have it, but if the steps ever need to be updated, it's easier not to have to change every number. ;)